rosdep/base.yaml

```diff
@@ -4617,6 +4617,11 @@ nkf:
   fedora: [nkf]
   gentoo: [app-i18n/nkf]
   ubuntu: [nkf]
+nlohmann-json:
+  debian: [nlohmann-json-dev]
+  fedra: [json]
+  gentoo: [nlohmann_json]
```

Comment:
I don't think this is the right format for Gentoo, but I'm not entirely sure.  Most of the other Gentoo keys seem to be of the sort `dev-cpp/nlohmann_json`.

Reply:
@allenh1

Reply:
> Most of the other Gentoo keys seem to be of the sort `dev-cpp/nlohmann_json`

@clalancette This is true. The reason behind this is that some packages have different categories but the same package name. For example, the ROS overlay uses `ros-[distro]/[package]` for the version of the package per ROS distribution (just to keep things organized).

*TL;DR*: this _would_ work, but I strongly prefer it be `dev-cpp/nlohmann_json` instead of just the bare `nlohmann_json`.

@hakuturu583 thanks for including the Gentoo keys in this patch! I very much appreciate it.

Reply:
Thanks! I fixed this issue in https://github.com/ros/rosdistro/pull/24432/commits/925e15528003f84a2c05d9cb8cebf59c1a5be4df